services/oauth/server.go

```diff
@@ -2,6 +2,9 @@ package oauth
 
 import (
 	"context"
+	"fmt"
+	"go-google-scraper-challenge/helpers"
+	"go-google-scraper-challenge/models"
 	"log"
 	"time"
```

Comment:
This is not following the [Go conventions](https://nimblehq.co/compass/development/code-conventions/golang/#imports) for Imports:

```suggestion
    "context"
	"fmt"
	"log"
	"time"
	
	"go-google-scraper-challenge/helpers"
	"go-google-scraper-challenge/models"
	
	"github.com/beego/beego/v2/server/web"
```

Reply:
@olivierobert fixed on [25e0826](https://github.com/carryall/go-google-scraper-challenge/pull/26/commits/25e0826c6b7bab0b58e47fc0f9a8d58c17db32d9)